src/main/java/org/folio/circulation/resources/RequestByInstanceIdResource.java

```diff
@@ -72,6 +75,7 @@
 
   private final Logger log;
   private static final String ITEM_ID_FIELD = "itemId";
```

Comment:
Maybe use `RequestProperties.ITEM_ID` instead of defining one here (I realize this code is not part of the PR, but just noticed it).

Reply:
Good point.